docs/en/fine_tuning/how_to/creatings.mdx

```diff
@@ -0,0 +1,125 @@
+---
+weight: 30
+---
+
+# Creating Fine-Tuning Tasks
+
+## Preparation
+
+### Datasets
+
+We support reading datasets from your S3 storage as well as from internal datasets.
```

Comment:
Alauda AI Fine Tuning tasks support reading datasets from S3 storage and Alauda AI datasets. You need to upload your datasets to S3 storage and Alauda AI datasets before creating fine tuning tasks.

> NOTE: dataset format should follow the need that the task templates metioned, e.g. yolov5 task template need the dataset formated as `coco128` like, and provide a YAML configuation file.